src/main/java/com/arpnetworking/metrics/mad/parsers/JsonToRecordParser.java

```diff
@@ -104,6 +106,8 @@ public Record parse(final byte[] data) throws ParsingException {
         final String version = versionNode.get().textValue().toLowerCase(Locale.getDefault());
         try {
             switch (version) {
+                case "2g":
+                    return parseV2gLogLine(jsonNode);
```

Comment:
My understanding is that the current Steno has evolved past what logback-steno supports. When I reached out about continued collaboration on the format there was no response from Groupon. Since this is now effectively a closed source format we can't continue to support it effectively in an open source project.

My plan is to fork the Steno format for logging in the next couple of months and deprecate its usage as a data container format in metrics. You are welcome to add an extension for parsing 2g wrapped in Steno but I would prefer that it be done closed source within Groupon (at least until the format is made public).